frontend/app/templates/components/badge-table.hbs

```diff
@@ -1,64 +1,70 @@
-<table class="ui sortable table">
-    <thead>
-        <tr>
-            <th>Badge Creator Name</th>
-            <th>Badge Name</th>
-            <th>Created At</th>
-            <th>Deleted At</th>
-            <th class="no-sort center aligned">Actions</th>
-        </tr>
-    </thead>
-    <tbody>
-        {{#each badges as |badge|}}
+{{#if empty}}
```

Comment:
I would be happy if you could properly align this

Reply:
I don't think there is indentation problem as it is approved by codacy, it checks for linting errors :)